base/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -89,15 +89,6 @@ spec:
           value: disable
         - name: CODEINTEL_PGUSER
           value: sg
-        # POD_NAME is used by CACHE_DIR
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        # CACHE_DIR stores larger items we cache. Majority of it is zip
-        # archives of repositories at a commit.
-        - name: CACHE_DIR
-          value: /mnt/cache/$(POD_NAME)
```

Comment:
We should also remove the volume mount and emptydir directive for it

Reply:
hah I spotted that as well, just pushed the update.